plugins/catalog-import/api-report.md

```diff
@@ -174,6 +172,16 @@ export interface EntityListComponentProps {
   withLinks?: boolean;
 }
 
+// @public (undocumented)
+type FormData_2 = {
+  title: string;
+  body: string;
+  componentName: string;
+  owner: string;
+  useCodeowners: boolean;
+};
+export { FormData_2 as FormData };
```

Comment:
This naming seems very unfortunate, probably best to find a more specific name for it?

Reply:
Yeah I tried to change as little as possible but since it looks "new" so to speak, then I guess we can take the opportunity.

Reply:
Yep it's only the "shape" that was exported for all of these things previously, now we start exporting the name as well